tests/integration/test_pip_packages.py

```diff
@@ -1,45 +1,47 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from os import path
+import pytest
 import tarfile
 
 import utils
 
 
-def test_pip_package_without_deps(test_env, tmpdir):
+@pytest.mark.parametrize("env_name", ["without_deps", "with_deps"])
+def test_all_pip_packages(env_name, test_env, tmpdir):
     """
-    Validate data in the pip package request without dependencies.
+    Validate data in the pip package request according to pytest env_name parameter.
 
     Process:
     Send new request to the Cachito API
     Send request to check status of existing request
 
     Checks:
     * Check that the request completes successfully
-    * Check that a single pip package is identified in response
+    * Check that expected pip packages are identified in response
```

Comment:
You should also check if there are proper dependencies listed in request:
without_deps: There are no dependencies under “.dependencies” 
with_deps: Dependencies are correctly listed under “.dependencies”

Reply:
@ejegrova PTAL ;)

Reply:
good job :-)